build/yamls/base/agent.yml

```diff
@@ -87,6 +87,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            # For using noEncap or hybrid traffic mode without AntreaProxy
+            - name: ALLOW_NO_ENCAP_WITHOUT_ANTREA_PROXY
+              value: "true"
```

Comment:
Not sure I understand the default value. It should be `false`, i.e. by default using noEncap without AntreaProxy should *not* be allowed

Or (even better IMO), we should not define this environment variable at all in the default manifest.

Reply:
Maybe we can solve it in one of these two ways:
1. Modify the default parameter to `false`.
2. Delete the environment variable in the default manifest, and then describe it in `docs/noencap-hybrid-modes.md`.

Reply:
We should delete it IMO